packages/icons/src/img/sprite.svg

```diff
@@ -98,6 +98,33 @@
         clip-rule="evenodd"
       />
     </symbol>
+    <symbol viewBox="0 0 24 24" id="lp-icon-arrow-connect-star">
+      <path
+        fill="#A34FDE"
```

Comment:
is this fill intentional?

Reply:
yeah, i believe we always want the star purple? since i don't think we otherwise have support for multi-color icons currently (but let me know if that's changed). i would expect the other path to be a different color dependent on light/dark mode

Reply:
Have an idea for allowing consumers to set path fills via CSS custom properties that we'll explore in the new year. Will let design decide how to proceed with this instance in the meantime.